src/com/dotmarketing/tag/business/TagFactoryImpl.java

```diff
@@ -567,6 +567,30 @@ public void deleteTagInodesByTagId(String tagId) throws DotDataException {
     }
 
     @Override
+    public void deleteTagInodesByInodeAndFieldVarName(String inode, String fieldVarName) throws DotDataException {
```

Comment:
I am just wondering what happen if one of the 

tagInodeCache.remove(tagInode);
tagCache.removeByInode(tagInode.getInode());

Throws an exception, could be finish with a dirty state?